ui/redux/rootReducer.js

```diff
@@ -277,15 +277,52 @@ const updateSavedVariant = (values, action = 'create') => {
 }
 
 export const updateVariantNote = (values) => {
-  if (values.variantGuid) {
-    return updateEntity(values, RECEIVE_DATA, `/api/saved_variant/${values.variantGuid}/note`, 'noteGuid')
+  if (values.familyGuids) {
+    if (values.variantGuid) {
+      return updateEntity(values, RECEIVE_DATA, `/api/saved_variant/${values.variantGuid}/note`, 'noteGuid')
+    }
+    return updateSavedVariant(values)
+  }
+  if (values[0].variantGuid) {
+    const variantGuids = []
+    let currVariantGuid
+    for (let i = 0; i < Object.keys(values).length; i++) {
+      try {
+        currVariantGuid = values[i].variantGuid
```

Comment:
Isn't `values` and array of objects? Couldn't this just be `values.map(({ variantGuid }) => variantGuid).filter(variantGuid => variantGuid)`